test/extended/cli/builds.go

```diff
@@ -0,0 +1,339 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	t "github.com/onsi/gomega/types"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"k8s.io/apimachinery/pkg/util/wait"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	"regexp"
+	"strings"
+	"time"
```

Comment:
Nit, but it's nice to split imports into the following logical groups:
1. built-in imports
2. 3rd party imports
3. openshift imports

You'll notice this being mostly applied to all of our source code and in k8s as well.